orb-slot-ctrl/tests/orb_slot_ctrl.rs

```diff
@@ -0,0 +1,77 @@
+use orb_slot_ctrl::test_utils::Fixture;
+use orb_slot_ctrl::{RootFsStatus, Slot};
+
+#[test]
+fn it_gets_current_slot() {
+    let fx = Fixture::new();
+    let slot = fx.slot_ctrl.get_current_slot().unwrap();
+    assert_eq!(slot, Slot::A)
+}
+
+#[test]
+fn it_gets_inactive_slot() {
+    let fx = Fixture::new();
+    let slot = fx.slot_ctrl.get_inactive_slot().unwrap();
+    assert_eq!(slot, Slot::B)
+}
+
+#[test]
+fn it_gets_and_sets_next_boot_slot() {
+    let fx = Fixture::new();
+    let slot = fx.slot_ctrl.get_next_boot_slot().unwrap();
```

Comment:
Why would it default to the next slot being B? I would assume A is the current, and the next slot remains A.
Is this a product of the mystery writes in the Fixture code?